### DISCUSSION TITLE
v2.0.0-alpha - sucessfull installation - inclomplete visualization

### DISCUSSION THREAD

**Discussion by MarkusWz** [kimai/kimai#3732] [On-Premise: Installation & Updates]
Hello,

I just have installed the alpha version of 2.0.0 on a test server to have a preview for the upcoming version.
I did a fresh installation but also an upgrade from an existing version.
There were no errors reported during the installation or upgrade.

Both ended up in the same issue:
The visualization of Kimai is not complete:
Some images/symbols are missing and I can't change the structure of the menu on the left side.
So I can't uncollaps the Administration menu but also not decollaps the Time Tracking folder for example.
I even can't log out or something, as a click on the user on the right does not have any result.

![grafik](https://user-images.githubusercontent.com/37211105/210572667-9b445d91-72cf-4184-90d2-924c47f83f03.png)

Any idea what is wrong?

Regards,
Markus

**Comment by kevinpapst** [comment]
The demo is installed exactly the same way as the installation docs say, so if the error is not present in the demo, it is likely an environment issue on your side.
Do you use Docker? 
Or a "regular" install? 
What does your browser console report?

**Reply by kevinpapst** [reply]
BTW: thank you for testing 👍

**Reply by MarkusWz** [reply]
I use a regular installation on Ubuntu 22.04.1 LTS for the test server.
PHP version is 8.2.0

Good idea to look in the browser:
Lookint to firefox web developer tool, I see that fonts are not found.
In addition it dosnt find 
GEThttp://10.21.20.42/build/runtime.f0079159.js
[HTTP/1.1 404 Not Found 48ms]
GEThttp://10.21.20.42/build/app.694c6cb5.js
[HTTP/1.1 404 Not Found 98ms]
GEThttp://10.21.20.42/build/app.73c27235.css

and in the end I get 
![grafik](https://user-images.githubusercontent.com/37211105/210598197-32ea3eee-4383-4f65-b95b-ea793c4e02ab.png)

**Reply by MarkusWz** [reply]
BTW: I always had issues with the native installation:
When people used the kimai folder the directory was listed.
http://10.21.20.42/kimai/
When I tryed to use the public folder
http://10.21.20.42/kimai/public/
the php was obviously read, but then the whole thing was not read properly.
So I created a index.php in the main folder of kimai
http://10.21.20.42/kimai/index.php
with the simple content of
<\?php header("Location: http://10.21.20.42/kimai/public/index.php"); ?>
This worked well then.

**Reply by kevinpapst** [reply]
That explains your problems. You have to map document root to public/ as described here:
https://www.kimai.org/documentation/webserver-configuration.html

**Reply by MarkusWz** [reply]
I'll review ... thanks!

**Reply by MarkusWz** [reply]
Works ... thanks!
